README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/tuura/fantasi.svg?branch=CI-travis)](https://travis-ci.org/tuura/fantasi)
```

Comment:
Do I have to set up CI at Travis or can you do it yourself?

Reply:
I did it myself, I didn't need any permissions :)